diagnostic.md

```diff
@@ -94,4 +92,4 @@ var teamMember = {
 What is the value of `teamMember['hobby']`?  Of `teamMember['name']`?  Of
 `teamMember['name']['surname']`?
 
-_Replace this text with your answer._
+exercise
```

Comment:
teamMember['hobby'] refers to the value of the "hobby" key of the teamMember object, which is "exercise".  teamMember['name'] refers to the object { given: 'Jason', surname: 'Weeks' }.  teamMember['name']['surname'] refers to the value ("Weeks") of the "surname" key in the nested object from the previous question.